<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>891</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="heart1">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="heart2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="heart3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameover">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>811</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>GAMEOVER</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreIs">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>430</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>score :</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>440</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
